Upstream: no (too lazy)
Reason: fixes build by getting rid of old cpp macro

diff --git a/src/fbdev.c b/src/fbdev.c
index f25ef72..02a2b7a 100644
--- a/src/fbdev.c
+++ b/src/fbdev.c
@@ -1010,7 +1010,7 @@ FBDevScreenInit(SCREEN_INIT_ARGS_DECL)
 	fPtr->CloseScreen = pScreen->CloseScreen;
 	pScreen->CloseScreen = FBDevCloseScreen;
 
-#if XV
+#if defined(XV)
 	{
 	    XF86VideoAdaptorPtr *ptr;
 

